src/BugsnagUnity/Native/Android/NativeThread.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using BugsnagUnity.Payload;
+using UnityEngine;
+
+namespace BugsnagUnity
+{
+    internal class NativeThread : NativePayloadClassWrapper, IThread
+    {
+        public NativeThread(AndroidJavaObject androidJavaObject) : base(androidJavaObject){}
+
+        public string Id { get => GetNativeLong("getId").ToString(); set => SetNativeLong("setId",long.Parse(value)); }
+
+        public bool? ErrorReportingThread => GetNativeBool("getErrorReportingThread");
```

Comment:
This might be fine as-is, but the desired behaviour here is to omit 'false' values from the payload - do you know if that'd be the current behaviour?